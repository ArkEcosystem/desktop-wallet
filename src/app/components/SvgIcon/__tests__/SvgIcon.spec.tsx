import React from "react";
import { render } from "@testing-library/react";

import { SvgIcon } from "../";

describe("SvgIcon", () => {
	it("should render", () => {
		const { container, asFragment } = render(<SvgIcon name="ark" />);

		expect(container).toBeTruthy();
		expect(asFragment()).toMatchSnapshot();
	});
});
