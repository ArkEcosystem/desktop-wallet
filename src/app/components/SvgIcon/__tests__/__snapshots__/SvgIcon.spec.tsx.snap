// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon should render 1`] = `<DocumentFragment />`;
